src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -712,18 +714,26 @@ static QString slurTieLineStyle(const SlurTie* s)
     QString lineType;
     QString rest;
     switch (s->styleType()) {
+    case SlurStyleType::Dashed:
+        lineType = "dashed";
+        break;
     case SlurStyleType::Dotted:
         lineType = "dotted";
         break;
-    case SlurStyleType::Dashed:
-        lineType = "dashed";
+    case SlurStyleType::Solid:
+        lineType = "solid";
```

Comment:
I'd rather let it fall trough here, not writing the "solid" at all, being the default